tests/test_validate.py

```diff
@@ -577,7 +577,7 @@ def test_no_expiration(self) -> None:
                 }
             ]
             issues = validate_test(sbom)
-            self.assertEqual(search_for_word_issues("expiration", issues), True)
+            self.assertEqual(issues, ["no issue"])
```

Comment:
What is the purpose of keeping this test and not removing it completely? It's not like we test for not required fields in general.